### DISCUSSION TITLE
Ynoa - gu10-cct converter

### DISCUSSION THREAD

**Discussion by AJKoster** [Koenkk/zigbee2mqtt#14318] [Devices]
I am trying to define an external converter for the Ynoa-gu10-cct (so not the RGB version which is already defined). I created the external converter file (see below) and included it in the configuration.xaml file as described in the guides. (I use the integration of zigbee2mqtt in homeassistant)

`
const fz = require('zigbee-herdsman-converters/converters/fromZigbee');
const tz = require('zigbee-herdsman-converters/converters/toZigbee');
const exposes = require('zigbee-herdsman-converters/lib/exposes');
const reporting = require('zigbee-herdsman-converters/lib/reporting');
const extend = require('zigbee-herdsman-converters/lib/extend');

const e = exposes.presets;
const ea = exposes.access;

const definition = {
    zigbeeModel: ['ZBT-CCTLight-GU100001'], // The model ID from: Device with modelID 'xxxxx' is not supported.
    model: 'LA-GU10-CCT', // Vendor model number, look on the device for a model number
    vendor: 'Ynoa', // Vendor of the device (only used for documentation and startup logging)
    description: 'Smart LED GU10 CCT', // Description of the device, copy from vendor site. (only used for documentation and startup logging)
    fromZigbee: [], // We will add this later
    toZigbee: [], // Should be empty, unless device can be controlled (e.g. lights, switches).
    extend: extend.light_onoff_brightness_colortemp({colorTempRange: [153, 454]}),

};

module.exports = definition;

`
The device is recognized and in the dashboard of zigbee2mqtt I get the right configuration (on/off state, brightness and colortemperature. But when i want to change e.g. the brightness, i get: "No converter available for 'brightness' (118)". Also the information presented in the dashboard is not right.
This is the first time i try to introduce a new device. Probably i make an obvious mistake or have to add additional information in the converter file, but whatever i tried, it did not resolve the problems. 
I hope someone can help me further... 
Thanks!

**Comment by Koenkk** [comment]
Try removing:

```
fromZigbee: [], // We will add this later
toZigbee: [], // Should be empty, unless device can be controlled (e.g. lights, switches).
```

**Reply by AJKoster** [reply]
Hi Koen
Great! Now it works. Thanks a lot!

**Reply by Koenkk** [reply]
Can you provide a link to this device?

**Comment by AJKoster** [comment]
The link is: https://www.ledlampenkopen.nu/led-lamp-gu10-ynoa-smart-home-zigbee-3-0-cct-dimbaar.html

**Reply by Koenkk** [reply]
Thanks, added the device!

Changes will be available in the dev branch in a few hours from now. (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)
